plugins/board-resources/src/components/popups/CardLabelsPicker.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import { Card, CardLabel } from '@anticrm/board'
+  import { Ref } from '@anticrm/core'
+  import { getClient } from '@anticrm/presentation'
+  import { Button, EditBox, Icon, IconEdit, IconCheck, Label, numberToHexColor, numberToRGB } from '@anticrm/ui'
+  import { isEmpty } from 'lodash'
+
+  import board from '../../plugin'
+  import { getBoardLabels } from '../../utils/BoardUtils'
+  import { updateCard } from '../../utils/CardUtils'
+
+  export let object: Card
+
+  const client = getClient()
+  let search: string | undefined = undefined
+  let labels = object.labels ?? []
+  let boardCardLabels: CardLabel[] = []
+  let filteredLabels: CardLabel[] = []
+  let hovered: Ref<CardLabel> | undefined = undefined
+
+  function applySearch() {
+    if (isEmpty(search)) {
```

Comment:
I think use lodash just for `isEmpty` is extra....

Reply:
Removed